### ISSUE TITLE
[Bug]: Tinted Badge in Teams Dark Theme does not meet accessibility requirements

### ISSUE THREAD

**Issue by emmayjiang** [microsoft/fluentui#23898]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
n/a
```

### Are you reporting Accessibility issue?

yes

### Reproduction

v9 themes

### Bug Description

![image](https://user-images.githubusercontent.com/31319479/178852706-bc2ebdd7-a01d-46e5-9a86-9c1c7e629481.png)
This is the tinted badge in Teams Dark Theme.
![image](https://user-images.githubusercontent.com/31319479/178852760-2711e369-3dad-463d-83da-e9de1651563d.png)
The color tokens involved are colorBrandForeground2 (the icon + text content within the tinted badge) and colorBrandBackground2 (the background color of the tinted badge). The WCAG requirement for text is 4.5:1, but this does not meet that requirement as it is 4.2:1.

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.

**Comment by daisygeng** [comment]
Hi, can we please update the slot in Teams dark theme to 120 to resolve the badge contrast? :) 
![image](https://user-images.githubusercontent.com/48726002/180488550-3708783c-698b-49fd-8f01-ecc99c03e35e.png)


fyi @miroslavstastny

**Comment by JustSlone** [comment]
@miroslavstastny we should follow up with Teams right away to confirm this change in the theme is feasible to make and make it before too long. Given this is fixing an accessibility issue we should fix the color, even though it's technically a visual breaking change.  Make sure this is called out clearly in the release notes. @jurokapsiar will follow up with positive handoff given the Urgent priority.

**Comment by miroslavstastny** [comment]
> Given this is fixing an accessibility issue we should fix the color, even though it's technically a visual breaking change.

To be honest, I consider any token changes proposed by designers to be fixes, not breaking changes.
